README.md

```diff
@@ -167,20 +174,20 @@ Provides an API Client for the Auth0 Management API (only methods meant to be us
 
 ```js
 var auth0 = new auth0.Management({
-  domain: "{YOUR_AUTH0_DOMAIN}",
-  token: "{ACCESS_TOKEN_FROM_THE_USER}"
+  domain: '{YOUR_AUTH0_DOMAIN}',
+  token: '{ACCESS_TOKEN_FROM_THE_USER}'
 });
 ```
 
 ### API
 
 - **getUser(userId, cb)**: Returns the user profile. https://auth0.com/docs/api/management/v2#!/Users/get_users_by_id
 - **patchUserMetadata(userId, userMetadata, cb)**: Updates the user metadata. It will patch the user metadata with the attributes sent. https://auth0.com/docs/api/management/v2#!/Users/patch_users_by_id
+- **patchUserMetadata(userId, user, cb)**: Updates the user attributes. It will patch the root attributes that the server allows it. To check what attributes can be patched, go to https://auth0.com/docs/api/management/v2#!/Users/patch_users_by_id
```

Comment:
the method is named the same as the one on the line above
```suggestion
- **patchUserAttributes(userId, user, cb)**: Updates the user attributes. It will patch the root attributes that the server allows it. To check what attributes can be patched, go to https://auth0.com/docs/api/management/v2#!/Users/patch_users_by_id
```

I also wonder given that in `user` you can pass `user_metadata`, if the old method should be deprecated and (eventually) replaced by this one.